_layouts/default.html

```diff
@@ -48,21 +48,41 @@
       </a>
     </div>
     <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav">
+      {% assign pages_top_size = site.html_pages
+          | where_exp:"item", "item.title != nil"
```

Comment:
Just to check - is this a breaking change behaviour-wise?

Reply:
I regard it as a bug fix. If a website using v0.3.3 has ordinary pages as well as (Just the Docs) collections, the ordinary pages are not displayed; after this PR, all ordinary pages will be displayed, as well as the collections.